scripts/run

```diff
@@ -14,4 +14,5 @@ mkdir -p build/testdata
 cd build/testdata
 export KUBECONFIG=
 export CATTLE_DEV_MODE=yes
+export PROXY_AUTHENTICATION=1
```

Comment:
Can this be any value? For example CATTLE_DEV_MODE just checks that _something_ is set

Reply:
It checks against "1", would be easy to set to anything. https://github.com/rancher/rancher/pull/21396/files#diff-b357539768154620e7a885bb7cc60911R453

Reply:
Fixed in primary PR.